modules/support_ticket/config/install/system.action.support_ticket_lock_action.yml

```diff
@@ -0,0 +1,9 @@
+id: support_ticket_lock_action
+label: 'Lock support ticket'
+status: true
+langcode: en
+type: support_ticket
+plugin: support_ticket_lock_action
```

Comment:
This is actually a support-ticket-specific feature. I got rid of "promote" and "sticky", but added "lock" (which is a way to lock a ticket so only admins can modify them). As this is not a concept in a node, I don't think it's actually generic.